test/cookbooks/php_test/recipes/opcache_apache.rb

```diff
@@ -0,0 +1,32 @@
+directory '/var/www/localhost' do
+  recursive true
+  owner 'apache'
+  group 'apache'
+end
+
+file '/var/www/localhost/index.php' do
+  content '<?php phpinfo(); ?>'
+  owner 'apache'
+  group 'apache'
+end
+
+package 'httpd'
```

Comment:
At this point, just use osl-apache for setting up apache in this test cookbook.

Reply:
yeah, circular dependency between osl-php and osl-apache `Failed to complete #converge action: [There is a circular dependency between osl-apache and osl-php`

Reply:
Oh derp, how about just using the plain apache2 cookbook then?

Reply:
Looks like the apache2 cookbook is a bigger issue somehow since it installs modphp5.4 by default, and it takes a few extra resources to get a vhost up and running with the proper modules. I can convert this to use apache2 if needed. I tried and hit the modphp issue and stopped there.

Reply:
AFAIK it shouldn't install modphp by default unless you're including a recipe which does that. I'll take a closer look later today if I have time.

Reply:
The mod_php is needed since the test cookbook sets up a vhost that returns the output of `<?php phpinfo() ?>` to test that both the apache interpreter and the CLI interpreter are using opcache